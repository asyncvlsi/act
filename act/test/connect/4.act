int x[4][4][4];
int x[4][2][7..8];

int y[5][4][4];

x = y;
