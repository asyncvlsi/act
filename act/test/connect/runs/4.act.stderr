In expanding ::<Global>
Connection: x = y
  LHS: int<32>[ [4][2][4]+[4][2][7..8]+[4][2..3][4] ]
  RHS: int<32>[5][4][4]
FATAL: Type-checking failed on connection
	Types `int<32>[ [4][2][4]+[4][2][7..8]+[4][2..3][4] ]' and `int<32>[5][4][4]' are not compatible
